src/qt/pivx/settings/settingswalletoptionswidget.cpp

```diff
@@ -65,6 +65,11 @@ void SettingsWalletOptionsWidget::onResetClicked()
     inform(tr("Options reset succeed"));
 }
 
+void SettingsWalletOptionsWidget::showEvent(QShowEvent *event)
+{
+    walletModel->loadWalletOptions();
```

Comment:
I'm a NACK for this change, commit 8ef86697f84627e48e22aeee8089b28d5b025c03.
It's the same change that was introduced in 2239, check my comment there https://github.com/PIVX-Project/PIVX/pull/2239#issuecomment-797758921

Reply:
Implemented signal-update and did some more cleanup.

Reply:
awesome 👌 👌